src/graphql-config/graphql-config.service.ts

```diff
@@ -0,0 +1,41 @@
+import type { ApolloDriverConfig } from '@nestjs/apollo';
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import type { GqlOptionsFactory } from '@nestjs/graphql';
+import { ApolloServerPluginLandingPageLocalDefault } from 'apollo-server-core';
+
+import type { EnvironmentVariables } from '../configuration/EnvironmentVariables';
+import { IBAN, Money } from '../shared/scalars';
+
+const mocks = {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  DateTime: () => {
+    return new Date();
+  },
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  String: () => {
+    return 'string-mock';
+  },
+};
+
+@Injectable()
+export class GraphQLConfigService implements GqlOptionsFactory {
+  constructor(private configService: ConfigService<EnvironmentVariables>) {}
+  createGqlOptions(): ApolloDriverConfig {
+    return {
+      playground: false,
+      debug: true,
+      mocks,
+      autoSchemaFile: true,
+      resolvers: { Money, IBAN },
+      plugins: [ApolloServerPluginLandingPageLocalDefault()],
+      introspection: this.configService.get('INTROSPECTION_ENABLED'),
+      //   cors: {
```

Comment:
why these lines are commented?

Reply:
It's a good question,
because of CORS :)
I wanted to test on my own, and forgot to uncomment it, 
we should do something to fix a problem on dev env

Reply:
What's the problem with CORS in dev env?

Reply:
In dev mode, testing manual Graphql is from Graphql studio, and there is a differentiation between origins ( our FE and ApolloServer Stuido)

Reply:
Let's add the GraphQL studio cors option conditionally on dev mode.